examples/seismic/tti/wavesolver.py

```diff
@@ -51,6 +47,20 @@ def op_fwd_save(self):
                                receiver=self.receiver, time_order=self.time_order,
                                space_order=self.space_order, **self._kwargs)
 
+    @cached_property
+    def op_fwd_legacy(self):
```

Comment:
again, you see, this is a combinational explosion of possibilities

Reply:
Different Kernel, different function, that was my idea

Reply:
This one is not changeable, as it is a cached property both need to exist or can't change kernel once the operator created